src/main/java/it/grid/storm/persistence/model/TapeRecallTO.java

```diff
@@ -451,4 +451,30 @@ public void setFakeRequestToken() {
 			log.error(e.getMessage(), e);
 		}
 	}
+
+	public enum TRequestType {
+
+		PTG_REQUEST("ptg"), BOL_REQUEST("bol"), BACK_REQUEST("back"), RECALL_REQUEST("rec");
+
+		private final String text;
```

Comment:
you don't need an additional string field if you use directy the enum name() method to get a string representation